app/grants/templates/grants/fund.html

```diff
@@ -35,62 +35,57 @@ <h2 class="fund__header">
-    {% csrf_token %}
-    <div class="row bottom-buffer">
-      <div class="form__group-horizontal">
-        <label class="form__label">{% trans "Contribution per Period" %}</label>
-        <div class="row">
-          <div class="col-4">
-            <input type="number" name="amount_per_period" class="form__input" required />
-          </div>
-          <div class="col-4">
-            <div class="form__select2">
-              <select class='js-select2' name='denomination' id='js-token'> </select>
+    <div class="container">
+      <div class="fund__title">{% trans "Fund Grant" %}</div>
+      <form action='{{ grant.pk }}' method="POST" id="js-fundGrant">
+        {% csrf_token %}
+        <div class="row bottom-buffer">
+          <div class="form__group-horizontal">
+            <label class="form__label">{% trans "Token Approval Limit" %}</label>
+            <div class="row">
+              <div class="col-8">
+                <input type="number" name="approve" class="form__input" required />
+              </div>
             </div>
           </div>
-        </div>
-      </div>
-      <div class="form__group-horizontal">
-        <label class="form__label">{% trans "MetaTx Gas Price" %}</label>
-        <div class="row">
-          <div class="col-8">
-            <input type="number" name="gas_price" class="form__input" required />
+          <div class="form__group-horizontal">
+            <label class="form__label">{% trans "Contribution Per Period" %}</label>
+            <div class="row">
+              <div class="col-4">
+                <input type="number" name="amount_per_period" class="form__input" required />
+              </div>
+              <div class="col-4">
+                <div class="form__select2">
+                  <select class='js-select2' name='denomination' id='js-token'> </select>
+                </div>
+              </div>
+            </div>
           </div>
-        </div>
-      </div>
-
-      <div class="col-4">
-        <div class="form__group-horizontal">
-          <button class="button button--primary button--full">{% trans "Contribute" %}</button>
-        </div>
-      </div>
-
-      <div class="col-4">
-        <div class="form__group-horizontal">
-          <!-- <label class="form__label">{% trans "Select period:" %}</label>
-          <div class="form__select2 fund__period">
-            <select name='period' id='period'>
-              <option value="month">{% trans "month" %}</option>
-              <option value="year">{% trans "year" %}</option>
-            </select> -->
-          <!-- </div> -->
-        </div>
-      </div>
-
-      <div class="col-4">
-        <div class="grant__progress">
-          <div class="progress">
-            <div class="progress-bar" style="width:50%"></div>
-            <div class="progress-bar progress-bar--fund" role="progressbar" style="width:10%">
-              <div class="contribution">
-                <span>10 ETH</span>
```

Comment:
Do we want to omit this?

Reply:
I dont think we do. Not sure exactly where it should be replaced.

Reply:
Added back in